src/classes/modules/java.base/java/lang/System.java

```diff
@@ -66,7 +67,7 @@ public class System {
     // would have prefered not to learn about. It's a mess WRT Java 1.5 / 6 compatibility
     // <2do> - most if this isn't supported yet
     SharedSecrets.setJavaLangAccess( createJavaLangAccess());
-
+    
```

Comment:
Revert this change (whitespace) so we can merge this change more easily with other branches in the future.